### PR TITLE
First Capstone project PR

### PR DESCRIPTION
**Pull request by Aidan-bn**
## In this PR the following has done

- I personalize the content of my page. 
- I choose a topic Rental Hall system
- I create two pages, home page and about page with two versions i.e mobile version and desktop version
- The home page have a link in the menu to the about page.
- The section "Featured speakers" has created dynamically in JavaScript
- Documentation the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
-    <link rel="stylesheet" href="/css/styles.css">
+    <link rel="stylesheet" href="./css/styles.css">
```
Comment (zieeco) [inline index.html:11]: :point_right: **NOTE: Please, I'd use this inline comment feature to highlight the proposed changes. Then changes are not related to this highlighted inline-block. Please follow closely:**

> while you've done well, I just want to point some things out that I observed about your PR which could be improved for your next projects:

> *Please, always merge the PR you've finished working to your base branch in order not to leave some confusion and also to ensure you don't submit a PR that has few commits which could make the review process somewhat difficult*

> **Reasons:** I see you have two PR of which you finished working on the `second-milestone` but didn't merge it to the main branch leaving you're main behind two branches. if you eventually merge this `final milestone`, you'd also have to merge the `second milestone` too in order to have the latest version of your project. please, be mindful of this. :pray:

Comment (zieeco) [reply index.html:11]: - Please, kindly update the file path as this I think you want to move to the root path (index.html) as this is making your page break. please update to this in your `pages` folder .. 
```
<div class="">
     <a href="../index.html">Welcome</a>
 </div>
```

Comment (zieeco) [reply index.html:11]: *`index.html` and `about.html`* desktop version.

- Please, kindly ensure both pages contain the same footer on the *desktop version*. Add the missing footer to the `about.html` page.

Comment (zieeco) [reply index.html:11]: **TOP NAV**

- Please, kindly fix the issue with the spacing on the top navbar:
- Use the correct background color.

*yours*
topnav

*template*
designN

Comment (zieeco) [reply index.html:11]: *h1-h6*

- Your page currently does not have the main header. You should have a single h1 on your page that denotes the main header content to improve the SEO of your website. kindly add one.

Comment (zieeco) [reply index.html:11]: *About paragraph*

- Please kindly add the missing surrounding border around your text to match the design as much as you can.

*yours*
db

*design*
bd

Comment (zieeco) [reply index.html:11]: *FONT SIZE*

- Please, kindly increase the font size of the text in your card to 16px below so the text content could be visible to those that are visually impaired thereby improving the accessibility of your page.

```
.main-contents li:last-child {
    font-size: 11px; - 16px;
}
```

*before*
fs

*after*
fss

Comment (zieeco) [reply index.html:11]: *javascript file*

- Please, let's keep our codebase clean, by removing unused code blocks. 
`  // hamburger.style.display = 'block';`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Aidan-bn,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Aidan-bn) [reply index.html:11]: Thanks for the comments. 
Resolved

Comment (Aidan-bn) [reply index.html:11]: For the footer section, the about page does not have the look as the index page from the template. Try to check the template.
Thanks

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Aidan-bn               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 

You have an amazing project! good job and I wish you the best of luck in the program.



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
